src/option.pxi

```diff
@@ -352,6 +352,8 @@ cdef class Options:
                 out[field.name] = (self_val, other_val)
         return out
 
+    logical_types: tuple[LogicalType] = LOGICAL_TYPES + (GzipStringType,)
```

Comment:
No type annotations needed, and LOGICAL_TYPES is a constant, just update the original definition in-place, no need to append it separately